app/controllers/projects_controller.rb

```diff
@@ -6,6 +8,13 @@ def index
     end
   end
 
+  def simple
+    @organization_id = params[:organization_id]
+    @projects = Project.where(projects_params).to_a
+    @released = @projects.select(&:fully_released?).sort_by(&:name)
+    @needs_work = @projects.reject(&:fully_released?).sort_by { |p| p.snapshots.length }
```

Comment:
Old snapshots are automatically cleaned up [here](https://github.com/joeyAghion/horizon/blob/2d2069ce49c1091935c6240cdf329c002914e0cb/app/services/comparison_service.rb#L66-L70), so most projects will have the same number. Maybe alphabetical still makes sense for this list?

Reply:
I'm actually interested in the number of commits, so this is fundamentally wrong :D